.github/workflows/release.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           goos: ${{ matrix.goos }}
           goarch: ${{ matrix.goarch }}
           goversion: "${{ env.GO_VERSION }}"
-          ldflags: -X 'github.com/conduktor/ctl/cmd.version=${{ github.event.release.tag_name }}' -X 'github.com/conduktor/ctl/cmd.hash=${{ github.sha }}'
+          ldflags: -X 'github.com/conduktor/ctl/utils.version=${{ github.event.release.tag_name }}' -X 'github.com/conduktor/ctl/utils.hash=${{ github.sha }}'
```

Comment:
update the Dockerfile too https://github.com/conduktor/ctl/blob/main/docker/Dockerfile#L8